### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by docprofsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Mandelbrot Fractal
+
+Renders the Mandelbrot set on a canvas with JavaScript.
+Rendering this set makes a fractal known as the Mandelbrot fractal.
```
Comment (docprofsky) [inline README.md]: Should there be a comma fractal? I think there should be one since the text after fractal is nonrestrictive (I think).

Comment (nedearb) [inline README.md]: Why do you have the second sentence? It doesn't add anything helpful.

Comment (docprofsky) [inline README.md]: Fixed.

Comment (nedearb) [inline README.md]: Again, what does the second sentence add?

Comment (docprofsky) [inline README.md]: I removed the second sentence; look at [7f94694](https://github.com/nedearb/mandelbrot/pull/1/commits/7f94694bd8139756b52fec029adb2f6621b57169)

Comment (nedearb) [inline README.md]: Oh sorry, I'm still learning pull requests.

**Pull request merged**